packages/core/src/dialects/postgres/query-generator-typescript.ts

```diff
@@ -77,6 +77,46 @@ export class PostgresQueryGeneratorTypeScript extends AbstractQueryGenerator {
     ]);
   }
 
+  getForeignKeyQuery(tableName: TableNameOrModel, columnName?: string) {
+    const table = this.extractTableDetails(tableName);
+
+    return joinSQLFragments([
+      'WITH unnested_pg_constraint AS (',
```

Comment:
could you add the comments but using JS comments?

Reply:
I've added the comment at the top now, I don't think the others are needed